src/main/java/jimmy/mcgymmy/logic/commands/ImportCommand.java

```diff
@@ -11,14 +11,17 @@
 import jimmy.mcgymmy.model.ReadOnlyMcGymmy;
 import jimmy.mcgymmy.storage.JsonMcGymmyStorage;
 
+/**
+ * Changes the data in mcgymmy to the data in the specified file.
+ * File must be available and in correct format
```

Comment:
Would be better if it is File must exist.

Reply:
Fixed